iogt/static/css/layout/_layout.scss

```diff
@@ -1,11 +1,12 @@
 // General container
 //__________________________________________________________
 .l-indent {
-  padding: 9px 24px;
+  padding: 10px 20px;
```

Comment:
I've tried to get rid of `l-indent` because it won't scale well when we do molo.core templates.